examples/tutorials/3d-map.py

```diff
@@ -0,0 +1,140 @@
+"""
+Creating a 3D map
+=================
+
+Plotting a three-dimensional map is handled by :meth:`pygmt.Figure.grdview`.
```

Comment:
This is not correct. Most plotting modules can plot 3D maps, for example https://www.pygmt.org/dev/gallery/plot/scatter3d.html.

Perhaps we can say:

> Create a 3-D perspective image

Reply:
Changed.